frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1134,6 +1144,10 @@ pub mod pallet {
 		SignedPhaseStarted { round: u32 },
```

Comment:
We could alternatively have an event `PhaseChange { old: PhaseType, new: PhaseType, round: u32}`. It seems a more clear way of showing transition of phases. Wdyt?

No strong opinion for either though.

Reply:
Yes, I like that. The encapsulating all events transition in one even is more future proof, especially if we need to add more phases (e.g. for the commit-reveal flow), so that the number of different events don't blow up.